vuln/core/README.txt

```diff
@@ -0,0 +1,631 @@
+server runs on an Intel CPU supporting AES-NI. This is a common configuration
+for TLS servers.
+
+The OpenSSL project has labelled this vulnerability high severity.
+
+--
+
+vulnerable: 4.x, 5.x, 6.x
+patched: ^4.4.4, ^5.11.1, ^6.1.0
+ref: https://github.com/nodejs/node/pull/6550
+ref: https://nodejs.org/en/blog/vulnerability/openssl-may-2016
+cve: CVE-2016-2105
+description: EVP_EncodeUpdate overflow
+ref: https://nodejs.org/en/blog/vulnerability/openssl-may-2016/#cve-2016-2107-padding-oracle-in-aes-ni-cbc-mac-check
+overview:
+An overflow can occur in the OpenSSL EVP_EncodeUpdate() function which is used
+for Base64 encoding of binary data. An attacker must be able to supply large
+amounts of input data in order to cause an overflow.
+
+Node.js uses the EVP_EncodeUpdate() internally during calls to
+crypto.Certificate#exportPublicKey() for SPKAC Certificate Signing
+Requests. User-supplied data must be passed to this method for
+applications to be vulnerable. This method has been available since
+Node.js v0.12.
+
+--
+
+cve: CVE-2016-1669
+vulnerable: >=6.0.0 <6.2.0, 5.x, 4.x
+patched: ^4.4.6, ^5.12.0
+ref: https://github.com/nodejs/node/commit/134c3b3977
+ref: https://nodejs.org/en/blog/vulnerability/june-2016-security-releases
+description: Buffer overflow in V8
+overview:
+Under certain conditions, V8 may improperly expand memory allocations in the
+Zone::New function. This could potentially be used to cause a Denial of Service
+via buffer overflow or as a trigger for a remote code execution.
+
+Although this bug is marked as high severity in the corresponding Chromium
+release (50.0.2661.102), our assessment is that this is low severity for
+Node.js users due to the level of difficulty in making use of this
+vulnerability. However, users are encouraged to upgrade their Node.js
+installation to ensure they are properly protected.
+
+--
+
+vulnerable: 5.x, 4.x, 6.x
+patched: ^5.12.0, ^4.5.0, ^6.2.1
+ref: https://github.com/nodejs/node/pull/7221
+ref: https://github.com/nodejs/node/pull/7562
+description: ignore negative allocation lengths
+
+--
+
+description: security flaw in the use of npm authentication tokens in HTTP requests
+vulnerable: 5.x, 4.x
+patched: ^5.10.0, 4.4.2
+ref: https://github.com/npm/node/pull/6
+overview:
+Upgrade npm to fixes a security flaw in the use of
+authentication tokens in HTTP requests that would allow an attacker to set up a
+server that could collect tokens from users of the command-line interface.
+Authentication tokens have previously been sent with every request made by the
+CLI for logged-in users, regardless of the destination of the request. This
+update fixes this by only including those tokens for requests made against the
+registry or registries used for the current install.
+
+This is a flaw in the version of npm included with node.
+
+npm is updated to 3.8.3 in node 5.10.1, and to 2.15.1 in node 4.4.2.
+
+--
+
+XXX split into two records?
```

Comment:
this seems like two related vulns, better to be two records with one CVE per record.